src/main/java/org/apache/maven/buildcache/BuildCacheMojosExecutionStrategy.java

```diff
@@ -139,7 +139,7 @@ public void execute(
                 }
             }
 
-            if (cacheState == INITIALIZED && (!restorable || !restored)) {
+            if (cacheState == INITIALIZED && (!restorable || !restored) && !skipCache) {
```

Comment:
Documentation page "Cache parameters" explicitly indicates : 
`-Dmaven.build.cache.skipCache : Skip looking up artifacts in caches. Does not affect writing artifacts to caches, disables only reading when set to true` with this usage scenario : `May be used to trigger a forced rebuild when matching artifacts do exist in caches`

So I was wondering if you go in a good direction?

Why exactly don't you add the "enabled" property in the project you want to "skip"?

Like : 
```xml
<properties>
    <maven.build.cache.enabled>false</maven.build.cache.enabled>
</properties>
```

In case this property fits your use case, I would suggest to delete this PR. And create a new one with a documentation update if you thing this property could be better documented.

Reply:
No worries, it improved my own knowledge on this extension. :)